### PR TITLE
This PR modifies some code formatting

### PR DESCRIPTION
**Pull request by aozeyu**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests.

## Purpose

Some description about what this PR wants to do.

## Approaches

Some description about how this PR achives the purpose. 

## Related Issues

_e.g._ Close #796

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

**Comment by aozeyu** [comment]
@BlockLiu @hk-lrzy

**Comment by aozeyu** [comment]
@hk-lrzy Some checks were not successful,please look ;

File: bitsail-common/src/main/java/com/bytedance/bitsail/common/util/FieldPathUtils.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public static Matcher listMatcher(String column) {
     return LIST_VALUE_PATTERN.matcher(column);
   }
 
-  public static enum PathType {
```
Comment (garyli1019) [inline bitsail-common/src/main/java/com/bytedance/bitsail/common/util/FieldPathUtils.java:64]: @aozeyu why do we remove this static

Comment (aozeyu) [reply bitsail-common/src/main/java/com/bytedance/bitsail/common/util/FieldPathUtils.java:64]: @garyli1019 hi, I think the properties of the enumeration java have already helped us turn it into static, there is no need to add static in front of the enumeration.
截屏2022-11-18 10 27 48

**Comment by aozeyu** [comment]
@hk-lrzy

File: bitsail-common/src/main/java/com/bytedance/bitsail/common/util/FieldPathUtils.java (hunk 1 above)
Comment (garyli1019) [reply bitsail-common/src/main/java/com/bytedance/bitsail/common/util/FieldPathUtils.java:64]: Got it, thanks for the information!

**Pull request merged**